extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -234,6 +234,22 @@ export class SubscriptionService {
 
       user().assert(this.pageConfig_, 'Page config is null');
 
+      if (this.viewer_.hasCapability('auth')) {
+        const serviceIds = ['local'];
+        this.platformStore_ = new PlatformStore(serviceIds);
+        this.viewer_.sendMessageAwaitResponse('auth',
+        /** @type {JsonObject} */({})).then(entitlementData => {
+          dev().assert(typeof entitlementData == 'object',
+              'entitlementData should be JsonObject');
+          const entitlement = Entitlement.parseFromJson(
+              /** @type {JsonObject}*/(entitlementData));
+          this.platformStore_.resolveEntitlement('local', entitlement);
```

Comment:
Not sure if it matters, but technically it'd be a "viewer" platform. I think we are ok to leave it as "local" - can't think of anything it'd break. But might be worth leaving comments here to explain.